CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
   the matching release version of the Grafana Agent instead of v0.14.0.
   (@rfratto)
 
+- [ENHANCEMENT] Switch to drone for releases. (@mattdurham)
```

Comment:
nit: I try to keep the CHANGELOG in type -> date order, can you add it as the last enhancement instead of the first?